lib-ee/emqx_ee_bridge/src/emqx_ee_bridge_mysql.erl

```diff
@@ -69,6 +69,7 @@ fields("config") ->
     [
         {enable, mk(boolean(), #{desc => ?DESC("config_enable"), default => true})},
         {direction, mk(egress, #{desc => ?DESC("config_direction"), default => egress})},
+        {sql_template, mk(binary(), #{desc => ?DESC("sql_template"), default => ?DEFAULT_SQL})},
```

Comment:
```suggestion
        {sql_template, mk(binary(), #{desc => ?DESC("sql_template"), default => ?DEFAULT_SQL, format => <<"sql">>})},
```
The `format` field can be used by the dashboard to provide a more suitable text box for SQL clauses.